### DISCUSSION TITLE
ETA on Patch Releases for JDK 11/17?

### DISCUSSION THREAD

**Discussion by kwheat2064** [microsoft/openjdk#313] [Q&A] [answered]
Hello,

Is there an ETA on a release of fixes for regressions discovered in OpenJDK 11/17?

Specifically [JDK Regression](https://mail.openjdk.org/pipermail/jdk-updates-dev/2022-August/016460.html)

Thanks!

**Comment by karianna** [comment, marked as answer]
The 11.0.16.1 and 17.0.4.1 releases are out :-) - https://docs.microsoft.com/en-gb/java/openjdk/download

**Reply by kwheat2064** [reply]
Thank you, @karianna
